Dockerfile

```diff
@@ -50,6 +50,8 @@ RUN apt-get update \
 
 # Install basic apt dependencies
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+    && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - >/dev/null \
```

Comment:
This will cause a long rebuild of the Dockerfile (once), but I guess it's better to do it now.

Reply:
Yeah, already found that :)